kimchi/src/verifier_index.rs

```diff
@@ -177,7 +179,7 @@ where
             max_poly_size: self.max_poly_size,
             max_quot_size: self.max_quot_size,
             powers_of_alpha: self.powers_of_alpha.clone(),
-            srs: Arc::clone(&self.srs),
+            srs: OnceCell::new(),
```

Comment:
If you change this to
```rust
srs: {
    let cell = OnceCell::new();
    cell.set(Arc::clone(&self.srs));
    cell
},
```
then we get back the old behaviour while still allowing for deserialisation.

Reply:
arr ok i gotcha!